src/main/java/com/dnsimple/data/DomainPrice.java

```diff
@@ -0,0 +1,37 @@
+package com.dnsimple.data;
+
+public class DomainPrice {
+    private final String domain;
+    private final Boolean premium;
+    private final Float registrationPrice;
```

Comment:
What do you folks think? Lets use float or change it to String?

Reply:
I am all in for Floats 👍  ‼️  Just be aware that we might break consistency across clients

Reply:
These actually need to be `Double`. The reference client implementation (the go client) uses `float64`, which is an IEEE-754 64-bit floating-point number. In Java, that would be `Double`. `Float` is an EEE-754 34-bit floating-point (I know, I know... bad naming Java!)

Reply:
Also, consider using the `double` primitives if they are not supposed to ever receive a `null` (also `boolean` for `premium` as well)